posthog/models/feedback/survey.py

```diff
@@ -56,7 +56,91 @@ class Meta:
     )
     type: models.CharField = models.CharField(max_length=40, choices=SurveyType.choices)
     conditions: models.JSONField = models.JSONField(blank=True, null=True)
-    questions: models.JSONField = models.JSONField(blank=True, null=True)
+    questions: models.JSONField = models.JSONField(
+        blank=True,
+        null=True,
+        help_text="""
+        The `array` of questions included in the survey. Each question must conform to one of the defined question types: Basic, Link, Rating, or Multiple Choice.
+
+        Basic (open-ended question)
+        - `type`: `open`
+        - `question`: The text of the question.
+        - `description`: Optional description of the question.
+        - `descriptionContentType`: Content type of the description (`html` or `text`).
+        - `optional`: Whether the question is optional.
```

Comment:
does our API documentation tool autogenerate types for these fields?  If not, I think we should specify the types.  It's not as important for fields that are `string` or `number`, especially when it's obviously a string/number field, but for something like this, would should make it clear that it's a boolean.  

Perhaps an example payload for a complex question could help?

Reply:
yeah, great point